src/mixins/TransformMixin.vue

```diff
@@ -0,0 +1,23 @@
+<script>
+import marked from 'marked';
+
+export default {
+  name:    'TransformMixin',
+  methods: {
+    /**
+     * Parse text to markdown format.
+     *
+     * @param {string} text - Text to parse.
+     *
+     * @return {string}
```

Comment:
If we pass null as `text` null will be returned. Is't it ?

Reply:
return type also should be updated